packages/bpk-component-calendar/src/BpkCalendarDate.tsx

```diff
@@ -193,7 +193,6 @@ class BpkCalendarDate extends PureComponent<Props> {
         type="button"
         style={style}
         className={classNames.join(' ')}
-        aria-hidden={isBlocked}
```

Comment:
I think this might be causing the blocked dates in the scrollable calendar to be read by the screen reader which doesn't seem to be happening on the main branch 🤔

Reply:
Agree with this point, currently the calendar wouldn't allow a AT user to go to dates that are in the past and make them inaccessible so with this removal this now allows them to select previous dates

Reply:
I guess are we happy with users being able to get to that date and with it mentioning 'dimmed' this is ok?

Reply:
Had a chat with Gert about this - it's not possible to keep this with the role of grid as it completely messes it up. Adding grid though brings a lot of benefits though like navigating up and down so even though you can now navigate through the blocked dates, it will be at most 30 days blocked, but you can just move down through the calendar so it's quick

Reply:
Yes, I purposefully unblocked the dimmed dates as otherwise, Grid navigation wouldn't work correctly :(